src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
is there a reason for multitfm here?

Reply:
The tasks for net6.0 only has 1 system.* dependency "System.Text.Json"  and `coverlet.msbuild.tasks.deps.json` uses net6.0 runtime. 

<img width="698" alt="image" src="https://github.com/coverlet-coverage/coverlet/assets/2315215/fd00acbd-c7bd-4a8d-8a98-d1df1ae0448e">

<img width="986" alt="image" src="https://github.com/coverlet-coverage/coverlet/assets/2315215/b9818f34-8b66-4c4c-af70-336641996135">

Reply:
Yes, this can be done. We only need to add target net6.0 and use a ItemGroup with condition for netstandard2.0.
This ItemGroup will have the PackageReference for System.Reflection.Metadata and System.Collections.Immutable for coverlet.collector (vstest compatibility).

Reply:
Does this also make sense for the `collector`?

Reply:
Yes, this can also be done for coverlet.collector 👍 but should be done in a separate PR. This requires an update of csproj 
 and coverlet.collector.targets file. I will create a PR after this one is merged (currently: framework 6.0).

> Extend DataCollector
The very first thing you will need to create is a Class Library project and add reference to Microsoft.TestPlatform.ObjectModel nuget package. Class Library project can target Desktop clr or dotnet core clr or both frameworks.